MonoGame.Framework/Utilities/OS.cs

```diff
@@ -0,0 +1,46 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace MonoGame.Utilities
+{
+    /// <summary>
+    /// Defines the operating system.
+    /// </summary>
+    public enum OS
```

Comment:
This is the check used by Protobuild, and it works fine for platform detection. There are no major Linux distributions that ship a Library folder. 

Having any platform specific code goes against the idea of DesktopGL, which is my point. The game can't actually have different references or use platform specific APIs (other than P/Invoke) in a DesktopGL executable. You can only really do cosmetic things like described above.